queries/QueryKeys.ts

```diff
@@ -127,3 +130,8 @@ export const pendingJoinRequestsQueryKey = (account: string) => [
   QueryKeys.PENDING_JOIN_REQUESTS,
   account,
 ];
+
+export const profileSocialsQueryKey = (
+  account: string,
+  peerAddress: string
+) => [QueryKeys.PROFILE_SOCIALS, account, peerAddress];
```

Comment:
Curious, why did you choose to add all keys into 1 file? Just really curious. I'm not doing that but maybe it's the best practice. What I do is put each keys into it's own file

Reply:
I just kinda did it one day, but I think how we are structuring folders by feature it probably makes more sense to split them into the separate files